recipes/assimulo/bld.bat

```diff
@@ -0,0 +1,7 @@
+
+:: --force-32bit=False --is_static=True
```

Comment:
What is this comment for? Is this left over code or do we need it for something?